ai/Gemini_ai_call_server.py

```
@@ -0,0 +1,846 @@
+        print(f"[서버] WebSocket 연결 종료: {session_key}")
+        if 'session_key' in locals():
+            await manager.disconnect(session_key)
+    except Exception as e:
+        print(f"[서버] WebSocket 오류: {str(e)}")
+        print(f"[서버] 오류 상세: {traceback.format_exc()}")
+        if 'session_key' in locals():
+            await manager.disconnect(session_key)
+        raise
+
+@app.post("/select_voice")
+async def select_voice(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)